src/common/Input/Input.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import clsx from 'clsx';
+
+function Input({
+	value,
+	onChange,
+	labelText = '',
+	placeholderText,
+	className = '',
+	type = 'text',
+	...rest
+}) {
+	return (
+		<div className='form-control w-full'>
+			{labelText ? (
```

Comment:
I made this way since not all "Inputs" must have "Label" and therefore  VScode throws an error when I don`t pass props. Or would it be better to pass an empty string as a props value? 
For "TextArea" I removed that logic.